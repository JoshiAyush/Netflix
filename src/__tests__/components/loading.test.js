import React from "react";
import { render } from "@testing-library/react";

import { LoadingContainer } from "../../components/index.js";

describe("<Loading />", () => {
  it("renders the <Loading /> component", () => {
    const { container, getByTestId } = render(
      <LoadingContainer.Picture
        src="/images/users/1.png"
        data-testid="loading-picture"
      />
    );

    expect(getByTestId("loading-picture")).toBeTruthy();

    expect(container.firstChild).toMatchSnapshot();
  });

  it("renders the <Loading.ReleaseBody />", () => {
    const { container } = render(<LoadingContainer.ReleaseBody />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
