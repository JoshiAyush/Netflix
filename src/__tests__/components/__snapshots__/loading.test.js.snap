// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders the <Loading /> component 1`] = `
<img
  class="style__Picture-sc-16p5sua-2 chCTTJ"
  data-testid="loading-picture"
  src="/images/users/1.png"
/>
`;

exports[`<Loading /> renders the <Loading.ReleaseBody /> 1`] = `null`;
